repository/src/main/resources/alfresco/repository.properties

```diff
@@ -752,8 +752,8 @@ encryption.ssl.truststore.keyMetaData.location=
 ## HttpClient config for Transform Service
 #enable mtls in HttpClientFactory for Transform Service.
 httpclient.config.transform.mTLSEnabled=false
-httpclient.config.transform.maxTotalConnections=40
-httpclient.config.transform.maxHostConnections=40
+httpclient.config.transform.maxTotalConnections=20
+httpclient.config.transform.maxHostConnections=20
```

Comment:
I'm adjusting the common config to the values from sharedFileStoreConnector, since those values here aren't even used properly due to how httpClients are created.